src/main/connector/system/Linux/Linux.yaml

```diff
@@ -120,12 +132,17 @@ monitors:
         source(1):
           # serviceName;load;active;sub;description
           type: osCommand
-          commandLine: systemctl list-units --type=service --all
+          commandLine: /usr/bin/systemctl list-units --type=service --all
           computes:
-            - type: awk
-              script: /\.service/ {sub(/^[^[:alnum:]]+\s*/, ""); sub("\.service", ""); printf $1 ";" $2 ";" $3 ";" $4 ";"; for (i=5; i<=NF; i++) printf "%s ", $i; printf "\n"}
+          - type: awk
+            script: |
+              /\.service/ {
+                sub(/^[^[:alnum:]]+\s*/, "");
+                sub("\.service", "");
+                printf $1 ";" $2 ";" $3 ";" $4 ";"; for (i=5; i<=NF; i++) printf "%s ", $i; printf "\n"

```

Comment:
In general, put each instruction on a separate line, just like in Java, C, etc.

```awk
printf($1 ";" $2 ";" $3 ";" $4 ";")
for (i=5; i<=NF; i++) {
  printf("%s ", $i)
}
printf("\n")
```

We also prefer calling functions (like `printf()`) with parenthesis. AWK allows function calling without parenthesis, but it can be confusing sometimes, imho.